lib/async_hooks.js

```diff
@@ -71,8 +71,7 @@ function fatalError(e) {
     Error.captureStackTrace(o, fatalError);
     process._rawDebug(o.stack);
   }
-  if (process.execArgv.some(
-      (e) => /^--abort[_-]on[_-]uncaught[_-]exception$/.test(e))) {
+  if (process.execArgv.some((e) => /^--abort[_-]on[_-]uncaught[_-]exception$/.test(e))) {
```

Comment:
This is inside `fatalError()` so I don't think RegExp recreation is an issue. If the RegExp passes, `process.abort()` is called. If the RegExp fails, `process.exit()` is called.